dashboard/app/models/plc/enrollment_module_assignment.rb

```diff
@@ -28,17 +29,16 @@ class Plc::EnrollmentModuleAssignment < ActiveRecord::Base
       COMPLETED = :completed
   ]
 
-  # The module status is completely determined by the completeness of its task assignments
   def status
-    tasks_to_complete = plc_task_assignments.joins(:plc_task).where.not('plc_tasks.type': 'Plc::LearningResourceTask')
-    completed_tasks = tasks_to_complete.where(status: Plc::EnrollmentTaskAssignment::COMPLETED)
+    levels_tracked = UserLevel.where(user: user, level: plc_learning_module.stage.script_levels.map(&:levels).flatten)
```

Comment:
`.pluck(:level_id)` won't work because the column is (usually) nil, but [`ScriptLevel#level_id` is still implemented](https://github.com/code-dot-org/code-dot-org/blob/59b53a7d5721136ed8d960d4d6ae17399f1a3e99/dashboard/app/models/script_level.rb#L52-L54).